app-frontend/src/app/login/login.js

```diff
@@ -0,0 +1,19 @@
+import angular from 'angular';
```

Comment:
There should only be [one component per file](https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#single-responsibility).

I like the setup from [here](https://github.com/gothinkster/productionready-angularjs-client/tree/master/src/js/article) personally. Seems like it would be a good application to model after potentially.